docs/node-operators/migration-guides/update-network-with-checkpoint.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Upgrade node with the checkpoint
+sidebar_label: Upgrade to latest
+sidebar_position: 1
+---
+
+## Assumptions for the upgrade guide
+The instructions below are written for Debian-like Linux operating systems. 
+
+The guide uses systemd commands(`systemctl` and `journalctl`) to control binaries in our setup. If you are using something different, that article's commands may vary.
+
+This guide is specifically intended for those who are already running a validator node with version `v0.68.0` or higher
+
+Before you start, note that the instructions use the following variables for file paths:
+
+* `<VEGA-NETWORK-HOME>`: the home path to the Vega network, e.g., `/home/vega/vega_home`
+* `<TENDERMINT-HOME>`: the Tendermint home path, e.g., `/home/vega/tendermint_home`
+* `<VEGAVISOR-HOME>`: the Vega Visor home path, e.g., `/home/vega/vegavisor_home`
+* `<BACKUP-FOLDER>`: the folder where you store backups, e.g., `/home/vega/backups`
+* `<VISOR-BIN>`: the path to the Vega Visor binary, e.g., `/home/vega/bin/visor`
+* `<VEGA-BIN>`: the path to the Vega core binary, e.g., `/home/vega/bin/vega`
+* `<CHAIN-ID>`: new chain ID for network; it is required to pass as an argument for data-node, e.g., current chain ID on mainnet is: `vega-mainnet-0009`
+
+The following are placeholders for the PostgreSQL connection details for the data node - the ones you put in the data node `config.toml`.
+
+* `<VEGA-DB-USER>` - PostgreSQL user you create and put in the data node config
+* `<VEGA-DB-NAME>` - PostgreSQL database name
+
+We will refer to the above paths in the following guide. The sample paths given above are just examples. We recommend setting the paths that align with the conventions adopted by your organisation.
+
+
+## Study the changes between versions
+Before upgrading your node software, read the upgrading file in the Vega repo for a full list of the changes between the two versions, and review the breaking API changes.
+
+Review the **[release notes](../../releases/overview.md)** for a list of breaking API changes for each version from 0.54 onwards.

```

Comment:
Maybe remove the ending telling about `0.53`?

Reply:
updated